readme.md

```diff
@@ -71,9 +77,11 @@ Returns the first filepath found (by respecting the order) or `null`.
 
 #### filename
 
-Type: `string`
+Type: `string` `Function`
```

Comment:
This is incorrect as it indicates that you can do `findUp([() => {}, () => {}], [options])`. Would be better to document it as another overload with different parameter name.

```md
### findUp(matcher, [options])

...

#### matcher

Type: `Function`
...
```